src/contracts/misc/overview.md

```diff
@@ -0,0 +1,39 @@
+This document describes the roles and responsibilities of the Gho Stewards.
+
+## Gho Steward Contracts
+
+These contracts each control different parameters related to GHO, GSM, and CCIP. They allow the Aave DAO and an approved Risk Council to change these parameters, according to set rules and configurations.
+
+Each Steward is designed to have a specific set of segregated responsibilities in an effort to avoid having to redeploy the entire original Steward. Instead, only the specific steward whose responsibilities are affected will have to be redeployed.
+
+### [GhoAaveSteward](src/contracts/misc/GhoAaveSteward.sol)
+
+This Steward manages parameters related to the GHO token. Specifically, it allows the Risk Council to change the following parameters:
+
+- Borrow Rate
+- Borrow Cap
+- Supply Cap
+
+In addition, the Aave DAO is allowed to change the configuration for the GHO Borrow Rate. This puts restrictions on how much the Risk Council is allowed to change parameters related to the borrow rate. There are 4 parameters that comprise the borrow rate:
+
+- optimalUsageRatio
+- baseVariableBorrowRate
+- variableRateSlope1
+- variableRateSlope2
+  For example, the Aave DAO can specify that the optimalUsageRatio variable may only be changed by 3% at a time.
```

Comment:
We may need to add a blank space here or something

Reply:
Done